src/server/main.cc

```diff
@@ -10,25 +10,42 @@
 #include <grpcpp/grpcpp.h>
 #include "src/proto/protocached.pb.h"
 #include "src/proto/protocached.grpc.pb.h"
+#include "src/server/data_store/data_store.h"
+#include "src/server/data_store/lru_cache_data_store.h"
 
 // TODO(joejoevictor): Move this to separate header and implementation file
 class ProtocachedServiceImpl final : public ::protocached::Protocached::Service {
+private:
```

Comment:
nit: style guide recommend private to declare after public/protected, as far as I remembered..